tests/kernel/kernel_info.cpp

```diff
@@ -44,15 +45,72 @@ TEST_CASE("Test kernel info", "[kernel]") {
 
   /** check program info parameters
    */
-  auto uint32Ret = kernel.get_info<sycl::info::kernel::num_args>();
-  check_return_type<uint32_t>(
-      uint32Ret, "sycl::kernel::get_info<sycl::info::kernel::num_args>()");
+  auto incorrect_num_args_usage = [&kernel] {
+    auto uint32Ret = kernel.get_info<sycl::info::kernel::num_args>();
+  };
+
+  {
+    /** According to SYCL specification info::kernel::num_args descriptor may
+       only be used to query a kernel that resides in a kernel bundle that was
```

Comment:
Just curious: is this what the current clang-format does? I would expect `only` aligned with `According`.